### DISCUSSION TITLE
Recailming Account<title>

### DISCUSSION THREAD

**Discussion by JoeTech89** [community/community#44785] [Enterprise] [answered]
### Select Topic Area

Product Feedback

### Body

   I have an account that I have created with Gitkraken that had created a GitHub account. It may have created it as an enterprise account, I haven't used it in a while. I want to move over to only using GitHub, I tried to recover the password and cant because it is not associated with and account, not a personal account or an enterprise account (I think that's the culprit). I then tried to create an account and I can't because there is one. Is there a way I can reclaim this account? All I've really found is claiming old usernames.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @JoeTech89 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
